docker/llm/finetune/qlora/cpu/docker/bigdl-qlora-finetuing-entrypoint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
This is for k8s, should not be combined with start script, and used in helm chart